docs/user/access-control/creating-sample-user.md

```diff
@@ -46,33 +46,14 @@ EOF
 
 Now we need to find token we can use to log in. Execute following command:
 
-For Bash:
-
 ```bash
-kubectl -n kubernetes-dashboard describe secret $(kubectl -n kubernetes-dashboard get secret | grep admin-user | awk '{print $1}')
-```
-
-For Powershell:
-```powershell
-kubectl -n kubernetes-dashboard describe secret $(kubectl -n kubernetes-dashboard get secret | sls admin-user | ForEach-Object { $_ -Split '\s+' } | Select -First 1)
+kubectl -n kubernetes-dashboard get secret $(kubectl -n kubernetes-dashboard get sa/admin-user -o jsonpath="{.secrets[0].name}") -o jsonpath="{.data.token}{'\n'}"
```

Comment:
Have you tested this in Powershell and can confirm that it works on Windows?

Reply:
I had Docker Desktop and tried it on PowerShell too. I did not set up the same namespace, but below a test run to show:

    PS C:\WINDOWS\system32> kubectl -n kube-public get secret $(kubectl -n kube-public get sa/default -o jsonpath='{.secrets[0].name}') -o jsonpath="{.data.token}{'\n'}"
    ZXlKaGJHY2lPaUpTVXpJMU5pSXNJbXRwWkNJNklrNUVlVEk1TjB4emFWZE5ZM2d6UWtwNGRIRktjSHBLU0V4dFR6QnNkbUptTVdwcWVteE1XRlJwYUZFaWZRLmV5SnBjM01pT2lKcmRXSmxjbTVsZEdWekwzTmxjblpwWTJWaFkyTnZkVzUwSWl3aWEzVmlaWEp1WlhSbGN5NXBieTl6WlhKMmFXTmxZV05qYjNWdWRDOXVZVzFsYzNCaFkyVWlPaUpyZFdKbExYQjFZbXhwWXlJc0ltdDFZbVZ5Ym1WMFpYTXVhVzh2YzJWeWRtbGpaV0ZqWTI5MWJuUXZjMlZqY21WMExtNWhiV1VpT2lKa1pXWmhkV3gwTFhSdmEyVnVMV1o2ZUdkbklpd2lhM1ZpWlhKdVpYUmxjeTVwYnk5elpYSjJhV05sWVdOamIzVnVkQzl6WlhKMmFXTmxMV0ZqWTI5MWJuUXVibUZ0WlNJNkltUmxabUYxYkhRaUxDSnJkV0psY201bGRHVnpMbWx2TDNObGNuWnBZMlZoWTJOdmRXNTBMM05sY25acFkyVXRZV05qYjNWdWRDNTFhV1FpT2lJME5ETmxOVE5qTUMwd00ySXdMVFEzWldVdFlqazJOeTAxWmpaaU9EWmpOelEyWVdZaUxDSnpkV0lpT2lKemVYTjBaVzA2YzJWeWRtbGpaV0ZqWTI5MWJuUTZhM1ZpWlMxd2RXSnNhV002WkdWbVlYVnNkQ0o5LnBYVGk5QWlPS28xcVZiWGNYSW85N0k2cTh6OFNYZ25LMlRmUFJ1Q0x4WmZWWHMzdjVGVXpUR1gwbnJrSWo5djljVWxvYmJWZm9UaVA0dzJmOHFwdEZpRi1obFpOWW1XVm5yb0M4R0xqd0xRS1VoZUdKU3BmLVNfNGVZTTdSaGhCSkRWSUJpLV9OSDRYTGRZeTIzUExRaUxrOTFHczlQNjFOQVVQSTRTYUVVSXZPWEFhSjF3S3hHSXhlaGlRMjZYMEtpU3hxeGw2cUluVUhCSFZ2ckpMa2NnYzZXV3EzVVZRRGltUkdUS1RyOEw3NlJranJocy0wU3pEMmZyc0Z1SjlPZkYydWJTNjMzYTRJcUxnNmQ1U2hjMW14eG51M1k0emRQbF9ZaWpXcjZ3RWFvTEVTVXJ2UTdKRDAzNVBHVVZUenRMQlY2X29CVWdWZS1BZnd2dVNyQQ==

Testing found the double quotes need to be outside because PowerShell does not preserve them inside single quotes. The `{'\n'}` is nice on bash, which prints the prompt immediately after program output without adding a newline.